templates/threatconnect.hbs

```diff
@@ -1467,50 +1470,190 @@
                     </div>
                   </div>
 
+                  {{! Add Integration Data Notes }}
+                  {{#if block.userOptions.enableAddingIntegrationData}}
```

Comment:
@ahaias-tc I think the left right padding was removed when you modified the scroll container to not include the submit buttons:

<img width="381" height="233" alt="image" src="https://github.com/user-attachments/assets/ff556066-cf76-49fe-9470-8eab8bca0f2a" />

Can you add padding back in on the "Add integratino data as note controls?